specification/compute/resource-manager/Microsoft.Compute/stable/2021-04-01/compute.json

```diff
@@ -9971,6 +9974,16 @@
       ],
       "description": "Describes a data disk."
     },
+    "CapacityReservationProfile": {
+      "type": "object",
+      "properties": {
+        "capacityReservationGroup": {
+          "$ref": "#/definitions/SubResource",
+          "description": "Specifies the capacity reservation group resource id that should be used for allocating the virtual machine provided enough capacity has been reserved. Please refer to https://aka.ms/CapacityReservation for more details."
```

Comment:
>Specifies the capacity reservation group resource id that should be used for allocating the virtual machine provided enough capacity has been res [](http://example.com/codeflow?start=26&length=145)

Specifies the capacity reservation group resource id that should be used for allocating the virtual machine or scaleset vm instances provided enough capacity has been reserved. Please refer to [https://aka.ms/CapacityReservation](https://aka.ms/CapacityReservation) for more details.